web/modules/custom/portland/modules/portland_zendesk/src/Element/PortlandSupportAgentWidget.php

```diff
@@ -72,6 +72,11 @@ public static function getCompositeElements(array $element) {
       '#title_display' => 'invisible',
       '#description' => 'Use this area as a scratch pad or as a field to add additional notes to the request. Anything submitted in this field will be included in the request description and may be visible to the requester.'
     ];
+    $element['escalate_issue'] = [
```

Comment:
Fixed the copy/paste error. I think the Escalate field is general enough that it might apply to any number of workflows. However, to avoid confusion, I've disabled it on all live forms except the ranger dispatch form.